src/main/java/com/thebois/views/toolbar/ToolbarView.java

```diff
@@ -0,0 +1,33 @@
+package com.thebois.views.toolbar;
+
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.ui.HorizontalGroup;
+
+import com.thebois.views.info.IActorView;
+
+/**
+ * The view that displays the different items in the toolbar.
+ */
+public class ToolbarView {
+
+    private final HorizontalGroup toolbar;
+
+    /**
+     * Instantiates a new view of a toolbar that the player can interact with.
```

Comment:
style

Nothing about this view allows for interaction, it's the different sub-views that do that.